src/dataflow/components/nodes/controls/plot-control.tsx

```diff
@@ -43,29 +43,55 @@ export class PlotControl extends Rete.Control {
     );
 
     const lineData = () => {
-      const data = {
+      const chartDataSets: ChartDataSets[] = [];
+
+      // determine how many datasets will be in graph
+      const recentValuesKey = "recentValues";
+      const recentValues: any = node.data[recentValuesKey];
+      const graphKeys: string[] = [];
+      if (recentValues && recentValues.length) {
+        Object.keys(recentValues[recentValues.length - 1]).forEach((objKey: any) => {
+          graphKeys.push(objKey);
+        });
+      }
```

Comment:
As per most recent request, ONLY show currently connected values.  The `recentValues` might contain disconnected (and now deleted) inputs.  For now, we will remove them.  As soon as an input is removed, it is removed from the graph.

Reply:
Should work, but we do need to make sure we check if `recentValues` is empty to handle the case where a user creates a node and immediately tries to view the graph before the heartbeat puts anything in `recentValues`.